crates/polars-time/Cargo.toml

```diff
@@ -24,13 +24,17 @@ regex = { workspace = true }
 serde = { workspace = true, features = ["derive"], optional = true }
 smartstring = { workspace = true }
 
+[dev-dependencies]
```

Comment:
The `test_rolling_group_by_aggs` test in `src/group_by/dynamic.rs` uses the `abs` function from `polars-ops`. The function is not used anywhere else in the crate and without specifying the `abs` feature the `polars-ops` crate builds without the `abs` function and the test fails to compile. Hence adding it to the dev-dependencies.

Reply:
We should then tag the test with a `cfg[feature = "abs"]`.

Reply:
Without the `dev-dependencies` the tests cannot find the `abs` method because it's not included in the `polars-ops` crate:
```
$ cargo test --all-features -p polars-time
   Compiling polars-ops v0.38.3 (/home/fila/workspace/polars/crates/polars-ops)
   Compiling polars-time v0.38.3 (/home/fila/workspace/polars/crates/polars-time)
warning: unused import: `polars_ops::prelude::*`
   --> crates/polars-time/src/group_by/dynamic.rs:672:9
    |
672 |     use polars_ops::prelude::*;
    |         ^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(unused_imports)]` on by default

error[E0425]: cannot find function `abs` in this scope
   --> crates/polars-time/src/group_by/dynamic.rs:792:17
    |
792 |         assert!(abs(&(var - expected)).unwrap().lt(1e-12).unwrap().all());
    |                 ^^^ not found in this scope
    |
help: use the `.` operator to call the method `polars_core::export::num_traits::Signed::abs` on `&polars_core::prelude::Series`
    |
792 |         assert!((&(var - expected)).abs().unwrap().lt(1e-12).unwrap().all());
```

Reply:
Right.. :+1: